ethstorage/p2p/cli/load_config.go

```diff
@@ -366,10 +366,12 @@ func loadSyncerParams(conf *p2p.Config, ctx *cli.Context) error {
 	maxRequestSize := ctx.GlobalUint64(flags.MaxRequestSize.Name)
 	syncConcurrency := ctx.GlobalUint64(flags.SyncConcurrency.Name)
 	fillEmptyConcurrency := ctx.GlobalInt(flags.FillEmptyConcurrency.Name)
+	maxPeers := ctx.GlobalInt(flags.PeersLo.Name)
```

Comment:
I think there should be a comment to note that since PeersLo is set to the maxPeers here, the PeersHi param is useless

Reply:
It sounds unnatural to me to set p2p.peers.lo as maxPeers.